aiohttp_session/__init__.py

```diff
@@ -1,7 +1,12 @@
 """User sessions for aiohttp.web."""
 
 import abc
-from collections import MutableMapping
+
+try:
+    from collections.abc import MutableMapping
```

Comment:
No need for `try/except` block here.
All supported python versions provide `collections.abc` module

Reply:
Ok, updated. Thanks.